data-source/census_individual.jsonnet

```diff
@@ -154,21 +152,21 @@ local confirmation = import 'blocks/confirmation.json';
           id: 'identity-and-health-group',
           title: 'Identity and Health',
           blocks: [
-            country_of_birth,
-            arrive_in_uk,
-            when_arrive_in_uk,
-            length_of_stay,
-            understand_welsh,
-            language,
+            country_of_birth(region_code),
+            arrive_in_uk(region_code),
+            when_arrive_in_uk(region_code),
+            length_of_stay(region_code),
+          ] + understandWelshBlock(region_code) + [
```

Comment:
Can't tell if I am happy with this or not.

Reply:
It's not pretty, but I can't think of anything better.